src/Playwright.Examples/Playwright.Examples.csproj

```diff
@@ -7,6 +7,7 @@
     <Nullable>enable</Nullable>
     <LangVersion>10</LangVersion>
     <NoWarn>CA2007</NoWarn>
+    <ReleaseVersion>0.0.0</ReleaseVersion>
```

Comment:
Not sure what this means, to set dll version we use <Version> <FileVersion> <AssemblyVersion> (list here https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generateassemblyinfo ) this is probably VS mac specific?

Reply:
Playwright.Examples is just an internal project which nobody does consume except we for testing sometimes. Should be fine then.